### ISSUE TITLE
Gemfile Checks get stuck

### ISSUE THREAD

**Issue by vzelenko** [manuelmeurer/railsbump#27]
# Problem Description

After uploading [THIS Gemfile](https://github.com/manuelmeurer/railsbump/files/4216122/Gemfile.txt), the Checks page ([EXACT URL](https://railsbump.org/gemfiles/78e398beda3d3208ccf2f2300cab9a802d99185b)) appears stuck in the state captured in images below.  No change in about 10-15 minutes.

## Part 1
![image](https://user-images.githubusercontent.com/8058697/74737017-f40b4880-5221-11ea-827b-7a21786eb358.png)

## Part 2
![image](https://user-images.githubusercontent.com/8058697/74737066-11401700-5222-11ea-9859-52e5d5093c60.png)

## Part 3
![image](https://user-images.githubusercontent.com/8058697/74737098-2321ba00-5222-11ea-8086-d1fc30cede1a.png)

## Part 4
![image](https://user-images.githubusercontent.com/8058697/74737149-45b3d300-5222-11ea-93f7-931dc4206102.png)

## Part 5
![image](https://user-images.githubusercontent.com/8058697/74737198-56644900-5222-11ea-8a37-38526a9be4d8.png)

**Comment by vzelenko** [comment]
Looks like the server might have crashed too.

**Comment by manuelmeurer** [comment]
@vzelenko Sorry, I don't know how to configure the db pool correctly, and I'm getting timeout errors left and right... deploys on Heroku take ages and when I try to open a console it says "Cannot run more than 1 free size dynos".
I wrote to Heroku to ask if they would sponsor the project with a paid dyno or two, and a Postgres db that alllows more than 10k rows, and they said no. 🤷

So I'm moving it off Heroku to a DigitalOcean dyno. They are very friendly to open-source projects and already sponsored another project of mine for a while (https://rubydocs.org/).

I disabled adding gems and gemfiles on RailsBump for now. Hopefully soon it will work properly.
I'm switching SuckerPunch with Sidekiq as well, which should be much more stable.
